support-frontend/knip.config.js

```diff
@@ -8,6 +8,26 @@ module.exports = {
 	entry: flattenedEntryPoints,
 	project: ['**/*.{js,jsx,ts,tsx,scss}!'],
 	ignoreExportsUsedInFile: true,
+	ignoreDependencies: [
+		// used in package.json
+		'@guardian/browserslist-config',
+		'lint-staged',
+		// used by scripts in package.json
+		'webpack-cli',
+		'webpack-dev-server',
+		// could be used to run Chromatic locally
+		'chromatic',
+		'concurrently', // used in devrun.sh
+		// used in webpack.common.js
```

Comment:
I hoped that knip should detect these but I can't seem to get it to work, the documentation implies it should 
https://knip.dev/explanations/plugins#webpack